.yarn/patches/@metamask-keyring-controller-npm-13.0.0-d94816a680.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/dist/KeyringController.js b/dist/KeyringController.js
+index fc649ea6fc97b905d811b236de638172fb10b548..beab676ab85e5e372eda7846e98b7d34af6317f5 100644
+--- a/dist/KeyringController.js
++++ b/dist/KeyringController.js
+@@ -1092,9 +1092,7 @@ _KeyringController_keyringBuilders = new WeakMap(), _KeyringController_keyrings
+ }, _KeyringController_addQRKeyring = function _KeyringController_addQRKeyring() {
+     return __awaiter(this, void 0, void 0, function* () {
+         // QRKeyring is not yet compatible with Keyring type from @metamask/utils
+-        const qrKeyring = (yield __classPrivateFieldGet(this, _KeyringController_instances, "m", _KeyringController_newKeyring).call(this, KeyringTypes.qr, {
+-            accounts: [],
+-        }));
++        const qrKeyring = (yield __classPrivateFieldGet(this, _KeyringController_instances, "m", _KeyringController_newKeyring).call(this, KeyringTypes.qr));
```

Comment:
Can you add a comment to this patch explaining the reason for it?

Reply:
i will add it now.